CHANGELOG.md

```diff
@@ -6,6 +6,10 @@ and this project adheres to [Semantic Versioning](http://semver.org/)
 
 ## [Unreleased]
 
+## [3.0.2] - 2020-03-13
+### Fixed
```

Comment:
I think 'added' is better not 'fixed'. It's not a bug it's a feature)

Reply:
You are right it is a feature.

And it is version `3.0.3` now.

Reply:
Or version 3.1.0 because of SemVer and this being a feature?